modules/identities/scripts/list.ts

```diff
@@ -0,0 +1,33 @@
+import { ScriptContext } from "../module.gen.ts";
+import { IdentityProviderInfo } from "../utils/types.ts";
+
+export interface Request {
+	userToken: string;
+}
+
+export interface Response {
+    identityProviders: IdentityProviderInfo[];
+}
+
+export async function run(
+	ctx: ScriptContext,
+	req: Request,
+): Promise<Response> {
+	await ctx.modules.rateLimit.throttlePublic({});
+
+    // Ensure the user token is valid and get the user ID
+    const { userId } = await ctx.modules.users.authenticateToken({ userToken: req.userToken } );
```

Comment:
we should let the caller authenticate the token and pass a raw user id. allows for better composability.

Reply:
This is a public script— are you sure we want that to be public by user ID?